docs/semgrep-assistant/overview.md

```diff
@@ -49,42 +49,42 @@ By categorizing your code through component tags, Semgrep Assistant can help you
+Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings when it identifies a true positive, if the rule doesn't have a human-written autofix. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, Assistant can recommend its use in the autofix suggestion for the issue in your code.
 
-You can set the minimum autofix confidence level required to display autofix suggestions from Semgrep Assistant.
+You can set the minimum autofix confidence level required to display autofix suggestions from Semgrep Assistant in the **Settings** page in Semgrep Cloud Platform.
 
-This feature is only active in PR and MR comments. Developers can review and verify Semgrep's generated fixes before they're applied.
+Autofixes are available in Semgrep Cloud Platform's **Findings page** under **Your code** in the [finding's details](/docs/semgrep-code/findings/#view-findings-details-about-a-specific-finding). They are also available in PR and MR comments, so developers can review and verify Semgrep's generated fixes before they're applied.
 
 ![Semgrep Assistant generating a potential fix](/img/semgrep-assistant-autofix.png)
 *Figure*. Semgrep Assistant generates a potential fix.
 
-You can view findings for which Assistant has generated autofix snippets in Semgrep Cloud Platform's **Findings page**. <!-- #10438 --> 
-
-![Sample finding entry with Semgrep Assistant verdict](/img/assistant-recommendations.png)
-
-The finding's detail page also includes a link to the PR or MR comment with the autofix; you can go to the PR or MR, where you can directly commit the autofix. <!-- #10516 --> <!-- This isn't unique to Assistant; it's available for autofix and PR/MR comments in general. -->
+The finding's details also include a link to the PR or MR comment with the autofix, so you can go to the PR or MR, where you can directly commit the autofix.
 
 ### Priority inbox
 
-Semgrep sends weekly emails with information on the top three backlog tasks across all according to Assistant. These results provided can include information for all Semgrep products that you have enabled. The emails are sent out on Monday to all organization admins.
+Semgrep sends weekly emails with information on the top three backlog tasks across all findings, according to Assistant. Unlike other Assistant features, these suggestions can include information for all Semgrep products that you have enabled. The emails are sent out on Monday to all organization admins.
```

Comment:
```suggestion
Semgrep sends weekly emails with information on the top three backlog tasks across all findings according to Assistant. Unlike other Assistant features, these suggestions can include information for all Semgrep products that you have enabled. The emails are sent out on Monday to all organization admins.
```

Reply:
I have the same issue with this sentence sans comma as with the other one, it's just too long to be strung together like that. How about this:

> Semgrep sends weekly emails with information on the Assistant's top three backlog tasks across all findings.

Reply:
That works